x/wasm/types/test_fixtures.go

```diff
@@ -2,14 +2,18 @@ package types
 
 import (
 	"bytes"
-	"crypto/sha256"
+	_ "embed"
 	"encoding/hex"
 	"encoding/json"
 	"math/rand"
 
+	wasmvm "github.com/CosmWasm/wasmvm"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+//go:embed testdata/reflect.wasm
+var wasmCode []byte
```

Comment:
it would be good to rename this to `reflectWasmCode` or something more verbose. There are some contracts used in this project